sw/airborne/subsystems/radio_control/sbus_common.c

```diff
@@ -54,6 +54,124 @@
 #endif
 
 
+
+/*
+ * S.bus decoder matrix. (extracted from PX4)
```

Comment:
I have updated the sbus2 branch. It now works for SBUS1, SBUS2 in 14CH mode and SBUS2 in 12CH mode with my R7008SB receiver and T14SG RC.
It is possible that some other end bytes will appear as it seems they are telling what is frame frame coming after the regular RC frame (voltage, gps, alti and so on). At some point we may wonder if we should care about this end byte or not as it is not a checksum and we don't use (for now) the other information.
@kevindehecker if you have the time would be nice if you can test it as well.